epinar/assignment1/q1.py

```diff
@@ -0,0 +1,30 @@
+
+def isAnagram(str1, str2, caseSensitive = False):
+	if not caseSensitive:
+		str1 = str1.lower()
+		str2 = str2.lower()
+
+	return (sorted(list(str1)) == sorted(list(str2)))
```

Comment:
The normalization logic is duplicated quite a few times: can you put it in a function and reuse that function for both strings in both exercices?

Reply:
Yes in a similar way to your answer.